src/app/tab/time/time.component.html

```diff
@@ -1,111 +1,289 @@
+      >
+        <circle cx="250" cy="11.4" r="11.4" />
+        <ellipse
+          transform="matrix(0.5 -0.866 0.866 0.5 147.13 341.5063)"
+          cx="369.3"
+          cy="43.3"
+          rx="11.4"
+          ry="11.4"
+        />
+        <ellipse
+          transform="matrix(0.866 -0.5 0.5 0.866 -4.1594 245.8406)"
+          cx="456.7"
+          cy="130.7"
+          rx="11.4"
+          ry="11.4"
+        />
+        <circle cx="488.6" cy="250" r="11.4" />
+        <ellipse
+          transform="matrix(0.5 -0.866 0.866 0.5 -91.5063 580.1427)"
+          cx="456.7"
+          cy="369.3"
+          rx="11.4"
+          ry="11.4"
+        />
+        <ellipse
+          transform="matrix(0.866 -0.5 0.5 0.866 -178.8533 245.8406)"
+          cx="369.3"
+          cy="456.7"
+          rx="11.4"
+          ry="11.4"
+        />
+        <circle cx="250" cy="488.6" r="11.4" />
+        <ellipse
+          transform="matrix(0.5 -0.866 0.866 0.5 -330.1427 341.5063)"
+          cx="130.7"
+          cy="456.7"
+          rx="11.4"
+          ry="11.4"
+        />
+        <ellipse
+          transform="matrix(0.866 -0.5 0.5 0.866 -178.8533 71.1467)"
+          cx="43.3"
+          cy="369.3"
+          rx="11.4"
+          ry="11.4"
+        />
+        <circle cx="11.4" cy="250" r="11.4" />
+        <ellipse
+          transform="matrix(0.5 -0.866 0.866 0.5 -91.5063 102.87)"
+          cx="43.3"
+          cy="130.7"
+          rx="11.4"
+          ry="11.4"
+        />
+        <ellipse
+          transform="matrix(0.866 -0.5 0.5 0.866 -4.1594 71.1467)"
+          cx="130.7"
+          cy="43.3"
+          rx="11.4"
+          ry="11.4"
+        />
       </svg>
-      <div class="analogCenter"
-        [style.background]="settings.config.design.foreground">
-      </div>
-      <div class="analogHour"
-        [ngStyle]="{'transform': getAnalogHour(clock.timezone)}"
-        [style.background]="settings.config.design.foreground">
+      <div
+        class="analogCenter"
+        [style.background]="settings.config.design.foreground"
+      ></div>
+      <div
+        class="analogHour"
+        [ngStyle]="{ transform: getAnalogHour(clock.timezone) }"
+        [style.background]="settings.config.design.foreground"
+      ></div>
+      <div
+        class="analogMinute"
+        [ngStyle]="{ transform: getAnalogMinute(clock.timezone) }"
+        [style.background]="settings.config.design.foreground"
+      ></div>
+      <div
+        class="analogSecond"
+        *ngIf="clock.seconds.enabled"
+        [ngClass]="{ dim: clock.seconds.dim }"
+        [ngStyle]="{ transform: getAnalogSecond(clock.timezone) }"
+        [style.background]="settings.config.design.foreground"
+      ></div>
+    </div>
+  </div>
+  <div
+    class="binary"
+    *ngIf="clock.binary.enabled"
+    [style.fontSize]="shared.getFontSize(clock.delimiter.scaling, 1.2)"
+  >
+    <div class="binaryContainer">
+      <div #hours id="hours">
```

Comment:
Please change `#hours` to be more specific to the binary clock so as not to be confusing in the future. Same with minutes and seconds

Reply:
Added binary to the front of h,m,s. and also updated the styles.